### PR TITLE
PBS cgroups hook throws TypeError 'dict' object is not callable

### PR DESCRIPTION
**Pull request by saimanikant**
#### Bug/feature Description
* PBS cgroups hook throws error TypeError ("'dict' object is not callable",)

#### Affected Platform(s)
* ALL

#### Cause / Analysis / Design
* use of “event.job_list()” for iterating is causing the error

#### Solution Description
* Using only “event.job_list” will make this dictionary iterator able 

#### Testing logs/output
* [test_after_fix_test_cgroup_periodic_update_check_values.txt](https://github.com/PBSPro/pbspro/files/2791670/test_after_fix_test_cgroup_periodic_update_check_values.txt)
[test_after_fix_test_cgroup_periodic_update_known_jobs.txt](https://github.com/PBSPro/pbspro/files/2791671/test_after_fix_test_cgroup_periodic_update_known_jobs.txt)
[test_before_fix_test_cgroup_periodic_update_check_values.txt](https://github.com/PBSPro/pbspro/files/2791672/test_before_fix_test_cgroup_periodic_update_check_values.txt)
[test_before_fix_test_cgroup_periodic_update_known_jobs.txt](https://github.com/PBSPro/pbspro/files/2791673/test_before_fix_test_cgroup_periodic_update_known_jobs.txt)

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by saimanikant** [comment]
[test_suite_after_fix_cgroup.txt](https://github.com/PBSPro/pbspro/files/2791686/test_suite_after_fix_cgroup.txt)

File: src/hooks/cgroups/pbs_cgroups.PY [hunk 1]
```diff
@@ -901,7 +901,7 @@ class HookUtils(object):
                                (caller_name(), msg))
         # Update the resource usage information for each job
         if cgroup.cfg['periodic_resc_update']:
-            for jobid in event.job_list():
+            for jobid in event.job_list:
```
Comment (mike0042) [inline src/hooks/cgroups/pbs_cgroups.PY]: Please add a comment above this line explaining that event.joblist is only iterable when the parenthesis are omitted.

**Comment by saimanikant** [comment]
All Tests have passed. Rebased and Squashed. PR is ready for merge.
Thanks!

**Comment by mike0042** [comment]
The last commit was not signed. Please sign it @saimanikant.

**Comment by saimanikant** [comment]
done.

**Pull request merged**